src/index.html

```diff
@@ -17,6 +34,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="header__logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link is-active"
+              href="1"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header__nav-item">
```